templates/about/power.html

```diff
@@ -42,10 +42,10 @@ <h4>Camper Vans &amp; Caravans</h4>
 <h4>Electric Vehicles</h4>
 <p>Owing to the increased price of providing electricity at EMF, and the increasing number of electric
     vehicles, we do not plan to provide generally-available electric vehicle charging at EMF 2022.
-    We ask that you arrive at EMF with enough charge to leave at the end of the event.
-</p>
+    We ask that you arrive at EMF with enough charge to leave at the end of the event.</p>
```

Comment:
This was a bit of a messy edit but leaving it in as it's actually more consistent with the rest of the page